lib/hammer_cli/main.rb

```diff
@@ -39,6 +39,9 @@ class MainCommand < AbstractCommand
     option ["--interactive"], "INTERACTIVE", _("Explicitly turn interactive mode on/off"),
       :format => HammerCLI::Options::Normalizers::Bool.new,
       :context_target => :interactive
+    option ["--only"], "FIELDS", _("Show selected columns or fields only"),
```

Comment:
Placing the option in the main command isn't very intuitive. Could you move it to the base info and list commands? It also doesn't make sense to use it for create, update and delete.